iphone/Classes/TiUIiOSProxy.m

```diff
@@ -840,5 +840,36 @@ -(id)createFeedbackGenerator:(id)args
 #endif
 #endif
 
+#if defined(USE_TI_UITEXTWIDGET) || defined(USE_TI_UITEXTAREA) || defined(USE_TI_UITEXTFIELD)
```

Comment:
`USE_TI_UITEXTWIDGET` is always defined when `USE_TI_UITEXTAREA ` or `USE_TI_UITEXTFIELD ` is defined, so I cannot think of a case where it is required to be checked manually.